### PR TITLE
[15.0][MIG] mrp_lot_on_hand_first

### PR DESCRIPTION
**Pull request by grindtildeath**
Migrated from:
 - [ ] #869 

Depends on:
 - [x] https://github.com/OCA/stock-logistics-workflow/pull/1098
 - [x] https://github.com/OCA/stock-logistics-workflow/pull/1099

### CODE REVIEW

**Review by ajaniszewska-dev — APPROVED** [review]

**Review by ivantodorovich — APPROVED** [review]

File: mrp_lot_on_hand_first/views/mrp_production.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            </field>
+            <xpath
+                expr="//field[@name='move_raw_ids']//field[@name='lot_ids']"
+                position="attributes"
+            >
+            <attribute
+                    name="context"
+                    operation="python_dict"
+                    key="name_search_qty_on_hand_first"
+                >parent.display_lots_on_hand_first</attribute>
```
Comment (ajaniszewska-dev) [inline mrp_lot_on_hand_first/views/mrp_production.xml]: just being curious, i see also lot of times construction with update - like below, why `python_dict` then?
```
<attribute name="context" operation="update">
          {"some_key": some_value}
</attribute>
```

Comment (grindtildeath) [reply mrp_lot_on_hand_first/views/mrp_production.xml]: It used to be so in version 14.0, but you're right that it has to use update in 15.0

**Comment by ivantodorovich** [comment]
Build is red 😢

**Pull request merged**